### DISCUSSION TITLE
simultaneous nebulization and intravenous drip administration

### DISCUSSION THREAD

**Discussion by Mannny-PK** [LAPKB/Pmetrics#113] [Q&A] [answered]
How do I define the absorption and central chambers when a patient is administered by multiple routes, such as simultaneous nebulization and intravenous drip administration?

#Primary
Ka, 0.1, 0.9
Ke, 0.001, 0.1
V, 30, 120
Tlag1, 0, 4

#Cov

WT

AFRICA

AGE

GENDER

HEIGHT

#Lag 
TLAG(1) = Tlag1

#Out
Y(1) = X(2)/V

#Err
G=5
0.02, 0.05, -0.0002, 0

**Comment by mhovd** [comment, marked as answer]
You can specify where a bolus dose will be administered with the #BOL block. Default behavior is for input 1 to go to compartment 1, etc. Per the [documentation for legacy Pmetrics](https://lapkb.github.io/Pmetrics/articles/models.html#specifying-models-in-legacy), this can be specified with
```
#BOL
NBCOMP(1) = 2
```

With the [R6 framework](https://lapkb.github.io/Pmetrics/articles/models.html#bolR6), it is similarly defined with `bol`, e.g.
```r
mod <- PM_model$new(list(
  bol = list("NBCOMP(1) = 2")
))
```

The above code is for bolus doses (DUR = 0), and will be delivered instantly. If you have infusions (DUR > 0), you can simply use `RATEIV(n)` where `n` is the input number for the administration pathway in your differential equations.

**Reply by Mannny-PK** [reply]
Many thanks for your help.
